understanding/contrast-minimum.html

```diff
@@ -135,236 +131,119 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
                         
                      </p>
                      
-                     <p>The 3:1 and 4.5:1 contrast ratios referenced in this Success Criterion are intended
-                        to be
-                        treated as threshold values. When comparing the computed contrast ratio to the Success
-                        Criterion
-                        ratio, the computed values should not be rounded (e.g., 4.499:1 would not meet the
-                        4.5:1 threshold).
+                     <p>この達成基準で言及されている 3:1 及び 4.5:1 のコントラスト比は、閾値として扱われることを意図している。算出されたコントラスト比を達成基準のコントラスト比と比較する場合、算出された値は丸めるべきではない（例えば、4.499:1 は 4.5:1 という閾値を満たさない）。
                      </p>
                      
                   </div>
                </div>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      
-                     <p>Because authors do not have control over user settings for font smoothing/anti-aliasing,
-                        when evaluating this
-                        Success Criterion, refer to the foreground and background colors obtained from the
-                        user agent, or the underlying
-                        markup and stylesheets, rather than the text as presented on screen.
+                     <p>フォントスムージング／アンチエイリアシングのユーザー設定をコンテンツ制作者はコントロールできないので、この達成基準を評価するときは、画面に表示されるテキストよりも、ユーザエージェントで取得できる、つまり基礎となるマークアップ及びスタイルシートから得られる、前景色と背景色を参照すること。
                      </p>
                      
-                     <p>Due to anti-aliasing, particularly thin or unusual fonts may be rendered by user agents
-                        with a much fainter
-                        color than the actual text color defined in the underlying CSS. This can lead to situations
-                        where text has
-                        a contrast ratio that nominally passes the Success Criterion, but has a much lower
-                        contrast in practice.
-                        In these cases, best practice would be for authors to choose a font with stronger/thicker
-                        lines,
-                        or to aim for a foreground/background color combination that exceeds the normative
-                        requirements
-                        of this Success Criterion.
-                        
+                     <p>アンチエイリアスが原因で、特に線の細いフォント又は特異なフォントは、ユーザエージェントによって、基礎となる CSS で定義された実際のテキストの色よりもずっと淡い色で表示されることがある。このため、テキストのコントラスト比が一応は達成基準を満たしていても、実際にはコントラストがかなり低いという状況が起こり得る。このような場合、ベストプラクティスは、コンテンツ制作者がより強く太い線を持つフォントを選ぶか、この達成基準の規範的要件を上回る前景色と背景色の組み合わせを目指すことであろう。
```

Comment:
anti-aliasingはアンチエイリアシングがアンチエイリアスのどっちかに揃えましょう（とりあえず前者に揃えました）
ほかにもいじくり回してみましたが、こんな感じ…？

```suggestion
                     <p>アンチエイリアシングが原因で、特に線の細いフォント又は特異なフォントは、ユーザエージェントによって、基礎となる CSS で定義された実際のテキストの色よりもずっとうすい色でレンダリングされることがある。このため、テキストのコントラスト比が名目上は達成基準を満たしていても、実際にはコントラストがかなり低いという状況が起こり得る。このような場合、ベストプラクティスは、コンテンツ制作者がより強く太い線を持つフォントを選ぶか、この達成基準の規範的要件を上回る前景色と背景色の組み合わせを目指すことである。
```